System.IO.Abstractions/IDirectory.cs

```diff
@@ -23,10 +23,14 @@ public interface IDirectory
         void Delete(string path, bool recursive);
         /// <inheritdoc cref="Directory.Exists"/>
         bool Exists(string path);
+
+#if !NETCOREAPP2_1
```

Comment:
Theses changes seem unrelated to me. Can you please remove all of them?

Reply:
some of the APIs have been removed in .net core 2.1 and because I've added netcoreapp2.1 as a new build target I had to exclude them.

Reply:
As mentioned above I’d prefer to have only .NET Standard targets in the core library

Reply:
I tried a second time and the test project won't build unless I remove the tests. I don't see any way around it.